openfl-workspace/flower-app-pytorch/src/connector.py

```diff
@@ -0,0 +1,43 @@
+import signal
+import sys
+from logging import getLogger
+from abc import ABC, abstractmethod
+
+class Connector(ABC):
+    """
+    Abstract base class for managing a server process of an external federated learning framework
+    and the connection with OpenFL's server.
+    """
```

Comment:
By the way, such `Connector` objects could be used for interacting with containerized _ML_ frameworks for better isolation from the core OpenFL components.